javascript/detail_and_listview.js

```diff
@@ -153,9 +149,10 @@ var createRouteCardSkeleton = function(object) {
 
     var divDistanceNumber = document.createElement('div');
     divDistanceNumber.setAttribute("class", "distancefrom-number");
-    divDistanceNumber.textContent = dbDistanceFromRef + ' mi';
+    var distanceFromValue = google.maps.geometry.spherical.computeDistanceBetween(walkerLocationTemp, dbStartLocationRef);
```

Comment:
definately calling APIs is part of the assignment, but there are actually formulas online that allow you to calculate the distance between two locations that also account for the radius of the earth.

anyways, fine for now, but I would try and localize it.